press/press/doctype/team/team.py

```diff
@@ -968,6 +968,12 @@ def process_stripe_webhook(doc, method):
 		process_micro_debit_test_charge(event)
 		return
 
+	if frappe.db.exists(
```

Comment:
This is fine but you should add unique validation so this doesn't happen anywhere else in future.

Is there ANY use case for duplicate intent id?

Reply:
> Is there ANY use case for duplicate intent id?

Not really, only reason I didn't add this earlier because we have non-unique records and we need to return 200 even if succeeded webhook is resent. Also stripe_payment_intent_id is only set in prepaid flow.

Added a custom validation anyway if somebody tries to mess up from desk https://github.com/frappe/press/commit/9f39b9a46ee6c58eca22cb3d786f871584c8ac5f